apps/src/code-studio/components/progress/StageLock.jsx

```diff
@@ -52,10 +52,12 @@ const StageLock = React.createClass({
     }
     return (
       <div style={{display: 'inline-block'}}>
+        /* className used only for purposes of uitest */
         <button
           style={progressStyles.blueButton}
           onClick={this.openLockDialog}
           disabled={this.props.saving}
+          className="uitest-locksettings"
```

Comment:
Interesting annotation, do you think we should adopt this more broadly?  I'd argue that it's not really necessary, since the test itself will fail if this classname is removed... but I guess It doesn't hurt :grin:.